test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -129,13 +129,10 @@ def test_apache_tomcat(image):
 )
 def test_sagemaker_studio_analytics_extension(training, package_name):
     framework, framework_version = test_utils.get_framework_and_version_from_tag(training)
-    utility_package_minimum_framework_version = {"pytorch": "1.7", "tensorflow": "2.4"}
-    utility_package_maximum_framework_version = {"pytorch": "1.8", "tensorflow": "2.6"}
-    
-    if framework not in utility_package_minimum_framework_version or Version(framework_version) < Version(
-        utility_package_minimum_framework_version[framework]) or Version(framework_version) > Version(
-        utility_package_maximum_framework_version[framework]
-    ):
+    studio_analytics_framework_version_limits = {"pytorch": ">=1.7", "tensorflow": ">=2.4"}
```

Comment:
does this also need to run for huggingface? I believe that function "get_framework_and_version_from_tag" will return "huggingface_pytorch" and "huggingface_tensorflow" for those cases